### PR TITLE
Add "gas" to the list of sensor.device_class

### PR DESCRIPTION
**Pull request by klacol**
## Proposed change

Add "gas" to the list of available device_classes for sensors to make it easy for users to add gas meters to the new energy dashboard, see also here: Based on this forum topic: https://community.home-assistant.io/t/adding-gas-to-energy-dashboard-solved/334988

## Type of change
    Docu fix

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @klacol, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19217).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/sensor.markdown [hunk 1]
```diff
@@ -23,6 +23,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **carbon_monoxide**: Carbon Monoxide in CO (Gas CNG/LPG)
 - **current**: Current in A.
 - **energy**: Energy in Wh or kWh.
+- **gas**: Gasvolume in m³
```
Comment (MartinHjelmare) [inline source/_integrations/sensor.markdown]:
```suggestion
- **gas**: Gasvolume in m³.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Review by klaasnicolaas — APPROVED** [review]
Thnx for the PR!

**Pull request merged**